workers/unity/Packages/com.improbable.gdk.deploymentlauncher/package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "com.improbable.gdk.deploymentlauncher",
+    "displayName": "SpatialOS GDK Deployment Launcher",
+    "version": "0.1.3",
+    "unity": "2018.2",
```

Comment:
2018.3?

Reply:
Oh. Yeah. Think I forgot to change all these in master 😬